addons/website/static/tests/tours/edit_menus.js

```diff
@@ -188,6 +204,10 @@ registerWebsitePreviewTour('edit_menus', {
         trigger: ".modal:not(.o_inactive_modal) .modal-footer button:contains(save)",
         run: "click",
     },
+    {
+        trigger: "body:not(:has(.oe_menu_editor))",
+    },
+    stepUtils.waitIframeIsReady(),
```

Comment:
we don t need waitIframeIsReady anymore (https://github.com/odoo/odoo/blob/a2bb8ca90d27da6dee93507e7f527304634c9a41/addons/web_tour/static/src/tour_service/tour_step_automatic.js#L120)